doop.c

```diff
@@ -635,6 +635,40 @@ Perl_do_trans(pTHX_ SV *sv)
     }
 }
 
+/* TODO: add API docs */
+STATIC bool
+S_sv_has_overload_method(pTHX_ SV *sv, int method)
```

Comment:
As this is a useful API function overall, I'd suggest adding it to `embed.fnc`, moving it to `gv.c` alongside some of the other magic-related functions, and making it not static.